packages/next/client/experimental-script.tsx

```diff
@@ -157,3 +184,9 @@ export default function Script(props: Props): JSX.Element | null {
 
   return null
 }
+
+Script.defaultProps = {
+  strategy: 'defer',
+}
```

Comment:
Instead of `defaultProps`, a default should be assigned in the object destructuring while reading props.